CMakeLists.txt

```diff
@@ -364,7 +364,7 @@ if(CRC32C_BUILD_TESTS)
     set_property(TARGET crc32c_capi_tests APPEND PROPERTY COMPILE_OPTIONS "/WX")
 
     # The Windows SDK version currently on CI produces warnings when some
-    # headers are #included using C99 compatibity mode or above. This workaround
+    # headers are #included using C99 compatibility mode or above. This workaround
```

Comment:
I think this line is over 80-characters now. Please wrap around to the next line?

Reply:
Fixed.